components/python/import-profiler/manifests/sample-manifest.p5m

```diff
@@ -14,7 +14,7 @@
 #
 
 set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
-set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=pkg.summary value="A basic python import profiler to find bottlenecks in import times."
```

Comment:
Sorry for this taking so long, but please, don't touch sample-manifest. It's there for the next one updating package could do gmake sample-manifest and look at git diff.